core/src/main/scala/com/avast/clients/rabbitmq/DefaultRabbitMQClientFactory.scala

```diff
@@ -551,6 +551,24 @@ private[rabbitmq] object DefaultRabbitMQClientFactory extends LazyLogging {
       }
   }
 
+  private def doTimeoutAction[A, F[_]: Effect](consumerConfig: ConsumerConfig,
+                                               timeoutsMeter: Meter,
+                                               e: TimeoutException): Task[DeliveryResult] = Task {
+    import consumerConfig._
+
+    timeoutsMeter.mark()
+
+    timeoutLogLevel match {
+      case Level.ERROR => logger.error(s"[$name] Task timed-out, applying DeliveryResult.${consumerConfig.timeoutAction}", e)
+      case Level.WARN => logger.warn(s"[$name] Task timed-out, applying DeliveryResult.${consumerConfig.timeoutAction}", e)
+      case Level.INFO => logger.info(s"[$name] Task timed-out, applying DeliveryResult.${consumerConfig.timeoutAction}", e)
+      case Level.DEBUG => logger.debug(s"[$name] Task timed-out, applying DeliveryResult.${consumerConfig.timeoutAction}", e)
+      case Level.TRACE => logger.trace(s"[$name] Task timed-out, applying DeliveryResult.${consumerConfig.timeoutAction}", e)
```

Comment:
As exactly the same string is repeated 4x, I would prefer to extract it to local variable.

Reply:
I thought someone will point this out :-)
However, this is intentionally like this - it's kind of optimization because the fact that you call `logger.warn` doesn't mean it will actually be used - based on result of `isWarnEnabled`.  
The only option (which I rejected originally but think it could work now) is to have the message as `lazy val` - would that be better?

Reply:
Fixed.